autoware_planning_msgs/srv/SetLaneChangeOverride.srv

```diff
@@ -0,0 +1,3 @@
+uint8 lane_change_direction
```

Comment:
Please define the detailed variables for the lane_change_direction (e.g. left, right, auto) as follows.
https://github.com/autowarefoundation/autoware_msgs/blob/f49c35c7a7b0110ad7bf5d91f67967833fd1c0d9/autoware_perception_msgs/msg/ObjectClassification.msg#L1-L12